3_source/3_models/1_adsEquilibrium/calcIsothermExtLang.m

```diff
@@ -196,9 +196,10 @@
         gConScaleFac = params.gConScaleFac;
         bC           = params.bC          ; 
         dimLessQsatC = params.dimLessQsatC;
+        gasCons      = params.gasCons;
             
         %Calcualte the dimensionless bC and qSatC
-        dimLessBC    = bC*gasCons*teScaleFac*gConScaleFac;            
+        dimLessBC    = bC.*gasCons*teScaleFac*gConScaleFac;            
```

Comment:
`.` was not needed here because all other terms except `bC` are scalars, and `bC` is the only vector here.